openshift/e2e-tests-openshift.sh

```diff
@@ -195,24 +256,32 @@ function run_e2e_tests(){
 
 function delete_istio_openshift(){
   echo ">> Bringing down Istio"
-  oc delete --ignore-not-found=true -f ${KNATIVE_ISTIO_CRD_YAML}
-  oc delete --ignore-not-found=true -f ${KNATIVE_ISTIO_YAML}
+  oc delete --ignore-not-found=true -f $ISTIO_RELEASE
+  oc delete --ignore-not-found=true -f $ISTIO_CRD_RELEASE
 }
 
 function delete_serving_openshift() {
   echo ">> Bringing down Serving"
-  oc delete --ignore-not-found=true -f ${KNATIVE_SERVING_RELEASE}
+  oc delete --ignore-not-found=true -f $SERVING_RELEASE
```

Comment:
I think this should be deleting the `serving-resolved.yaml` file. AFAIK, $SERVING_RESOLVED is not defined. We have the same problem in knative-serving.